packages/shared-schemas/src/error-constants.ts

```
@@ -0,0 +1,47 @@
+    'The compute service was not found. Run `compute list` to see available services.',
+  CHECK_DOCKER_IMAGE:
+    'Check the Docker image URL is valid and accessible. Ensure the image exists in the registry.',
+  CHECK_FLY_CAPACITY:
+    'Compute service deployment failed. Check the Fly.io region has capacity, or try a different region.',
+  RETRY_COMPUTE_OPERATION:
+    'The operation failed due to a transient error. Wait a moment and try again.',
+
+  // Add more next actions as needed
+} as const;
+
+export type NextActionKey = keyof typeof NEXT_ACTION;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)